cv_next/app/feed/components/feed.tsx

```diff
@@ -1,19 +1,26 @@
 "use client";
 
 import CVItemLink from "@/app/feed/components/CVItemLink";
-import { useCallback, useContext, useEffect, useRef, useState } from "react";
+import { useCallback, useContext, useEffect, useMemo, useRef, useState } from "react";
 import CVItem from "./CVItem";
 import { CvsContext, CvsDispatchContext } from "@/providers/cvs-provider";
 import { ReloadButton } from "@/components/ui/reloadButton";
 import Definitions, { API_DEFINITIONS } from "@/lib/definitions";
 import { useInView } from "react-intersection-observer";
-import { FilterPanel } from "@/app/feed/components/filterPanel";
+import { CATEGORY_PARAM, DESCRIPTION_PARAM, FilterPanel } from "@/app/feed/components/filterPanel";
 import ReactLoading from "react-loading";
 import { filterValues } from "@/types/models/filters";
 import { useApiFetch } from "@/hooks/useAPIFetch";
 import { ScrollToTop } from "@/components/ui/scrollToTop";
+import { useSearchParams } from "next/navigation";
+import { toCategoryNumber } from "@/lib/utils";
+import { useDebounceCallback } from "@/hooks/useDebounceCallback";
 
 export default function Feed() {
+  const searchParams = useSearchParams();
+  const uriCategories = searchParams.get(CATEGORY_PARAM);
+  const description = searchParams.get(DESCRIPTION_PARAM);
```

Comment:
Considering the fact that this is presented to the user, is there a chance that this(and possibly other user-based inputs) are prone to XSS or sqli?

Reply:
Hmm.. This is being treated as String and there's no `eval` on it; it's also not used directly to be sent to the server (as there's the whole string-to-id transformation), so I can't think of an obvious way this can be abused. But lmk if you think there's any reason to do some special parsing here.

Reply:
As regards to other user-based inputs - sure, this should be on a per-case basis how to treat the information. We can check e.g. if supabase doesn't do anything on their side to resolve sqli-type attacks (for e.g. search functionality), but because it's essentially an orm I'm pretty sure they do.

Reply:
Ok so it's handled on their end
Resolving this